solutions/banking/main.tf

```diff
@@ -15,51 +15,51 @@ resource "random_string" "unique_identifier" {
 data "ibm_iam_auth_token" "tokendata" {}
 
 # Resource group - create if it doesn't exist
-# module "resource_group" {
-#   source  = "terraform-ibm-modules/resource-group/ibm"
-#   version = "1.1.5"
-#   # If an existing resource group is not set (null), then create a new one
-#   resource_group_name          = var.resource_group_name == null ? local.unique_identifier : null
-#   existing_resource_group_name = var.resource_group_name
-# }
+module "resource_group" {
+  source  = "terraform-ibm-modules/resource-group/ibm"
+  version = "1.1.5"
+  # If an existing resource group is not set (null), then create a new one
+  resource_group_name          = var.resource_group_name == null ? local.unique_identifier : null
+  existing_resource_group_name = var.resource_group_name
+}
 
 # create COS instance for WatsonX.AI project
-# module "cos" {
-#   source            = "terraform-ibm-modules/cos/ibm//modules/fscloud"
-#   version           = "7.5.3"
-#   resource_group_id = module.resource_group.resource_group_id
-#   cos_instance_name = "gen-ai-rag-sample-app-cos-instance"
-#   cos_plan          = "standard"
-# }
+module "cos" {
+  source            = "terraform-ibm-modules/cos/ibm//modules/fscloud"
+  version           = "7.5.3"
+  resource_group_id = module.resource_group.resource_group_id
+  cos_instance_name = "gen-ai-rag-sample-app-cos-instance"
+  cos_plan          = "standard"
+}
+
+# create CD service for toolchain use
+resource "ibm_resource_instance" "cd_instance" {
```

Comment:
@brendankellyibm  the `cd_instance` is also created as part of the prereq for the tests where toolchains are also created. 

Do we need `cd_instance` here as part of the DA? We are not creating any toolchains in the DA itself.

Reply:
Did we also reach out to ALM DA team to see why they don't support creating the service?

Reply:
I'm creating it here until we have an option to provision the CD service in the ALM DA - but in any case, they won't be doing it before Think, so we can revisit it afterwards.

@rajatagarwal-ibm it is required to have the CD service to run the toolchains created previously.